Automattic/jetpack
# Search: Implicitly activate search when widget added

Fixes #8763

#### Changes proposed in this Pull Request:

If the user's plan supports search, then show the widget even if the search module/functionality is disabled. If the widget is used, then enable the search module automatically.

#### Testing instructions:

Make sure your test site has a plan that supports Jetpack Search.

1. Remove any active search widgets and disable the search module via Settings -> Traffic.
2. Verify that the widget still shows up in both the admin area and the customizer.
3. Add the widget to your sidebar.
4. Load the frontend of your site and verify that the widget shows up and works.
5. Verify that the search module is now enabled again.
